pandas/tests/plotting/test_hist_method.py

```diff
@@ -565,6 +565,33 @@ def test_hist_secondary_legend(self):
         assert ax.get_yaxis().get_visible()
         tm.close()
 
+    def test_hist_weights(self):
```

Comment:
Can you clarify test name?

Reply:
I've changed it to `test_hist_with_nans_and_weights`, is this clear enough?